docs/reference.md

```diff
@@ -258,4 +266,54 @@ applied if the marked test method fails or errors.
 Log a summary of the status of the model. This is automatically called before the model
 is cleaned up.
 
+#### `async def track_model(self, alias: str, model_name: Optional[str] = None, cloud_name: Optional[str] = None, create: bool = True, **kwargs,) -> Model`
+
+Indicate to `ops_test` to track a new model which is automatically created in juju or an existing juju model referenced by model_name. 
+This allows support for `ops_test` to track multiple models on various clouds by a unique alias name.
+
+#### `async def forget_model(self, alias: str, timeout: Optional[Union[float, int]] = None)`
+
+Indicate to `ops_test` to forget an existing model.
+* If this model was intended to be kept, the model will be forgotten by the test structure immediately
+* If not, the model will attempt to be destroyed and removed from the controller.  
+
+A Timeout Exception will be raised if a timeout is provided and the model isn't destroyed within that period
+
+
+#### `def model_context(self, alias: str) -> Generator[Model, None, None]:`
+
+The only way to switch between tracked models is by using this method to change
+the context of the model to which the tests refer.   
+
+For example, assume there are two models being tracked by ops_test: "main" and "secondary"
+The following test would `PASS` due to the nature of `model_context`'s function.
+
+```python
+def test_second_model(ops_test):
+    assert ops_test.current_alias == "main"
+    primary_model = ops_test.model
+    with ops_test.model_context("secondary") as secondary_model:
+        # now this is testing the secondary model
+        assert ops_test.current_alias == "secondary"
+        assert ops_test.model == secondary_model
+
+    # now this is testing the main model
+    assert secondary_model != primary_model
+    assert ops_test.current_alias == "main"
+    assert ops_test.model == primary_model
+```
+
+The following ops_test properties will change their values within a `model_context`
+* `current_alias`
+* `tmp_path`
+* `model_config`
+* `model`
+* `model_full_name`
+* `model_name`
+* `cloud_name`
+* `keep_model`
```

Comment:
Are there cases where you'd want to interact with both primary and secondary contexts at the same time?  I guess in that case you'd use `ops_test.models['secondary'].something`, etc?

Reply:
Yes, correct you could do either of these

```python
ops_test.models["main"].model.do.things()
ops_test.models["secondary"].model.do.things()
````

or

```python
primary = ops_test.model 
with ops_test.model_context("secondary") as secondary:
   pass

primary.do.things()
secondary.do.things()
```